frontend/javascripts/oxalis/model/volumetracing/volumelayer.js

```diff
@@ -173,8 +180,8 @@ class VolumeLayer {
   }
 
   getContourList(useGlobalCoords: boolean = false) {
-    const volumeTracing = enforceVolumeTracing(Store.getState().tracing);
-    const globalContourList = volumeTracing.contourList;
+    const globalContourList = getVolumeTracingById(Store.getState().tracing, this.volumeTracingId)
+      .contourList;
```

Comment:
This looks weirdly formatted, but maybe that's the way to do it and I'm too tired :sweat_smile:

Reply:
I guess, we have to trust prettier on this :smile: